infra/consumer/main.tf

```diff
@@ -0,0 +1,22 @@
+#make bucket
```

Comment:
Why is this commented out? As a good practice, let's try to avoid commented out code.

Reply:
that's not code. It's just a comment.